crates/anvil/core/src/eth/mod.rs

```diff
@@ -359,6 +359,10 @@ pub enum EthRequest {
     #[serde(rename = "anvil_setBalance", alias = "hardhat_setBalance")]
     SetBalance(Address, #[serde(deserialize_with = "deserialize_number")] U256),
 
+    /// Modifies the ERC20 balance of an account.
+    #[serde(rename = "anvil_dealERC20", alias = "hardhat_dealERC20")]
```

Comment:
I think we can also add an alias for the tenderly name, 

unsure about the response type though